transport/http/src/test/scripts/org/kaazing/gateway/transport/http/security/auth/client.forbidden.rpt

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property connectAddress "http://localhost:8000/resource"
+property someCredentials ${http:loginBase64Encoder("user:password")}
+property authHeader ${http:append("Basic ", someCredentials)}
+
+connect ${connectAddress} 
```

Comment:
It would be better if all of these scripts came from the HTTP k3po spec or another spec.